recipe/meta.yaml

```diff
@@ -104,18 +127,20 @@ outputs:
       commands:
         - python -m TBB -h
         - python -m tbb -h
-        - python -m tbb test                             # [linux]  # XXX fix it
+        - python -m tbb test
     about:
       summary: TBB module for Python
       license: Apache 2.0
-      dev_url: https://github.com/01org/tbb
+      dev_url: https://github.com/oneapi-src/oneTBB
 
 about:
-  home: http://www.threadingbuildingblocks.org
+  home: https://github.com/oneapi-src/oneTBB
   license: Apache-2.0
-  license_file: LICENSE
+  license_file:
+    - LICENSE.txt
+    - third-party-programs.txt
   summary: High level abstract threading library
-  dev_url: https://github.com/01org/tbb
+  dev_url: https://github.com/oneapi-src/oneTBB
   doc_url: https://software.intel.com/en-us/node/506039
```

Comment:
The doc_url needs to be updated: https://software.intel.com/en-us/oneapi-tbb-documentation